model-optimizer/extensions/middle/ConcatOptimization.py

```diff
@@ -112,8 +108,19 @@ def find_and_replace_pattern(self, graph: Graph):
                     continue
                 shape = in_port.data.get_shape()
                 assert shape is not None
-                if np.array_equal(shape, [0]):
+                if 0 in shape:
                     in_port.disconnect()
 
             connected_input_ports = [in_port for in_port in concat.in_ports().values() if not in_port.disconnected()]
             assert len(connected_input_ports), 'Concat {} does nothing'.format(concat.soft_get('name', concat.id))
```

Comment:
I believe this message is confusing. Could you please correct it?

Reply:
Done